src/Scientist/Internals/Experiment.cs

```diff
@@ -30,6 +31,14 @@ public ExperimentInstance(string name, Func<Task<T>> control, Func<Task<T>> cand
             _candidate = candidate;
         }
 
+        public ExperimentInstance(string name, Func<Task<T>> control, Func<Task<T>> candidate, Func<T, T, bool> resultComparison)
```

Comment:
Seems like `resultComparer` would be a better name. To me, "comparison" sounds like the result of comparing. Thoughts?